src/Libraries/CoreNodeModels/ColorRange.cs

```diff
@@ -20,7 +20,7 @@ namespace CoreNodeModels
     [NodeSearchTags("ColorRangeSearchTags", typeof(Resources))]
 
     [InPortNames("colors", "indices", "value")]
-    [InPortTypes("Color[]", "double[]", "double")]
+    [InPortTypes("DSCore.Color[]", "double[]", "double")]
```

Comment:
NodeModel nodes need to specify the fully qualified types in both the `InPortTypes` and `OutPortTypes` attributes for node autocomplete to work properly. We need to make this a guideline for NodeModel node authors if they wish for node autocomplete to work with their nodes. Without this requirement, it is difficult to get the feature working for such nodes.

Reply:
Agreed